### ISSUE TITLE
Autocomplete suggestions not visible

### ISSUE THREAD

**Issue by rholdos** [mallowigi/material-theme-ui-lite#1]
When I start typing the autocomplete window with suggestions pop up, but I can't see the suggestions probably because text color is the same as background color. When i press arrow down or arrow up, suggestions became visible. Any suggestions on how to fix this behaviour?

![Screenshot 2019-09-04 at 14 00 51](https://user-images.githubusercontent.com/34940346/64253031-aa050380-cf1c-11e9-9e1d-c6a4da88b794.png)
![Screenshot 2019-09-04 at 14 01 14](https://user-images.githubusercontent.com/34940346/64253032-aa9d9a00-cf1c-11e9-9f74-7fb3e66e9cc1.png)

Sometimes I am also getting this error, I guess this may be the problem.
```java.lang.NullPointerException
	at com.mallowigi.idea.themes.MTAbstractThemeAction.getThemeName(MTAbstractThemeAction.java:73)
	at com.mallowigi.idea.themes.MTAbstractThemeAction.isSelected(MTAbstractThemeAction.java:60)
	at com.intellij.ide.util.gotoByName.GotoActionModel$GotoActionListCellRenderer.getListCellRendererComponent(GotoActionModel.java:758)
	at com.intellij.ide.actions.searcheverywhere.SearchEverywhereUI$CompositeCellRenderer.getListCellRendererComponent(SearchEverywhereUI.java:962)
	at com.intellij.ui.ExpandedItemListCellRendererWrapper.getListCellRendererComponent(ExpandedItemListCellRendererWrapper.java:38)
	at com.intellij.ui.ExpandedItemListCellRendererWrapper.getListCellRendererComponent(ExpandedItemListCellRendererWrapper.java:38)
	at com.intellij.ui.components.WideSelectionListUI.updateLayoutState(WideSelectionListUI.java:167)
	at java.desktop/javax.swing.plaf.basic.BasicListUI.maybeUpdateLayoutState(BasicListUI.java:1389)
	at java.desktop/javax.swing.plaf.basic.BasicListUI$Handler.valueChanged(BasicListUI.java:2785)
	at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:219)
	at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:199)
	at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:246)
	at java.desktop/javax.swing.DefaultListSelectionModel.changeSelection(DefaultListSelectionModel.java:443)
	at java.desktop/javax.swing.DefaultListSelectionModel.changeSelection(DefaultListSelectionModel.java:453)
	at java.desktop/javax.swing.DefaultListSelectionModel.removeSelectionIntervalImpl(DefaultListSelectionModel.java:614)
	at java.desktop/javax.swing.DefaultListSelectionModel.clearSelection(DefaultListSelectionModel.java:458)
	at java.desktop/javax.swing.JList.setSelectedIndices(JList.java:2226)
	at com.intellij.ide.actions.searcheverywhere.SEListSelectionTracker.restoreSelection(SEListSelectionTracker.java:73)
	at com.intellij.ide.actions.searcheverywhere.SearchEverywhereUI$SearchListener.elementsAdded(SearchEverywhereUI.java:1569)
	at com.intellij.ide.actions.searcheverywhere.ThrottlingListenerWrapper.lambda$null$0(ThrottlingListenerWrapper.java:38)
	at com.intellij.ide.actions.searcheverywhere.ThrottlingListenerWrapper.lambda$new$2(ThrottlingListenerWrapper.java:38)
	at com.intellij.ide.actions.searcheverywhere.ThrottlingListenerWrapper$Buffer.flush(ThrottlingListenerWrapper.java:131)
	at com.intellij.ide.actions.searcheverywhere.ThrottlingListenerWrapper.searchFinished(ThrottlingListenerWrapper.java:69)
	at com.intellij.ide.actions.searcheverywhere.MultiThreadSearcher$FullSearchResultsAccumulator.lambda$searchFinished$6(MultiThreadSearcher.java:404)
	at com.intellij.ide.actions.searcheverywhere.MultiThreadSearcher$ResultsAccumulator.lambda$runInNotificationExecutor$1(MultiThreadSearcher.java:256)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:312)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:433)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:817)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:405)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:404)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)```

**Comment by Davoleo** [comment]
Same issue on Light Owl theme

**Comment by manulaiko** [comment]
I can confirm it happens with every theme, https://youtrack.jetbrains.com/issue/IDEA-223245

**Comment by mallowigi** [comment]
Will be fixed in version 5.0
